form_app/lib/src/autofill.dart

```diff
@@ -14,6 +14,32 @@ class AutofillDemo extends StatefulWidget {
 class _AutofillDemoState extends State<AutofillDemo> {
   final _formKey = GlobalKey<FormState>();
 
+  FocusNode _lastname, _email, _telephone, _streetAddresss, _postalCode, _country, _countrycode;
```

Comment:
nit: We generally use a max line length of 80 in this repo, and typically break fields out onto their own lines, even if the types are the same:

```dart
FocusNode _lastName;
FocusNode _mail;
... etc. ...
```